src/styles/Base/variables.scss

```diff
@@ -1,10 +1,13 @@
+/* Fonts */
 $font-text: 'Roboto Condensed', sans-serif;
 $font-heading: 'PT Serif', serif;
 $font-links: 'Montserrat', sans-serif;
 
 /* Colors */
-
 $black: #1F1C17;
 $yellow: #FFA10C;
 $brown: #593C1A;
 $white: #FAFAFA;
+
```

Comment:
That is not a problem. If you merge your PR before me I will resolve the conflicts or the other way around, so don't be worried :+1: